### PR TITLE
Implementation of dark mode in the storybooks

### LINKED ISSUE #699: Implement dark mode for our storybooks
## Description
We could have a dark mode for our 3 storybooks. There are many plug-ins available for the same.

- https://storybook.js.org/docs/react/configure/theming
- https://github.com/hipstersmoothie/storybook-dark-mode

(The above links are for reference sake only)

What is needed, perhaps, is a button which toggles state from light to dark thereby changing the theme of the entire storybook. 

## Implementation

- [ ] I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by neeraj-2**
## Fixes

Fixes #699 by @nimishbongale 

## Description
Implements toggling b/w Dark mode and Light mode in the 3 storybooks `Fonts`, `Vocabulary` and `Vue-Vocabulary` 

## Demo:

ezgif com-gif-maker

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: packages/fonts/.storybook/manager.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import { addons } from '@storybook/addons'
 
-import theme from './theme'
+import { themes } from '@storybook/theming';
+import 'storybook-dark-mode/register';
```
Comment (dhruvkb) [inline packages/fonts/.storybook/manager.js:4]: This line [belongs to `addons.js`](https://github.com/hipstersmoothie/storybook-dark-mode#installation).

**Review by dhruvkb — COMMENTED** [review]
Can you check why the build is breaking? This would be easier to merge if we could see the theme live on Netlify.

**Comment by neeraj-2** [comment]
@nimishbongale can u please review why some tests are failing?

**Comment by dhruvkb** [comment]
I think it's because you haven't committed the changes to `package.json` and `package-lock.json` so the new plugin wasn't found.

**Comment by dhruvkb** [comment]
And @neeraj-2, you can see the build logs directly, by clicking the 'cross' icon and then checking the 'Details' link:
Screenshot 2020-10-13 at 11 50 53 AM

**Comment by neeraj-2** [comment]
> And @neeraj-2, you can see the build logs directly, by clicking the 'cross' icon and then checking the 'Details' link:
> Screenshot 2020-10-13 at 11 50 53 AM

thanks, @dhruvkb, I will make the required changes.

**Comment by nimishbongale** [comment]
> @nimishbongale can u please review why some tests are failing?

@neeraj-2

I think @dhruvkb correctly pointed it out 👍 

Also can we have the toggle switch tab button (having the moon icon) at the end of the tab bar? 

Just like how it is [here](https://github.com/hipstersmoothie/storybook-dark-mode/blob/master/example.gif)?

**Comment by neeraj-2** [comment]
I have created a new draft PR #723 for the same .So closing this one.

**Pull request closed without merging**